Blog - Istio sessions at Kubecon NA 2024 (#15857)
istio/istio.io: content/en/blog/2024/kubecon-na/index.md

---

> title: "Istio in Salt Lake City! See you at KubeCon North America 2024"

Reviewer: ```suggestion
title: "Istio in Salt Lake City!"
```

description: Celebrate Istio at KubeCon + CloudNativeCon North America 2024.

> publishdate: 2024-11-01

Reviewer: ```suggestion
publishdate: 2024-11-05
```

attribution: "Faseela K, for the Istio Steering Committee"
keywords: [Istio Day,Istio,conference,KubeCon,CloudNativeCon]
---

> Amazing lineup of Istio activities await you at [KubeCon + CloudNativeCon North America 2024 in Salt Lake City, Utah](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)!

Reviewer: ```suggestion
An amazing lineup of Istio activities awaits you in  in Salt Lake City, Utah at [KubeCon + CloudNativeCon North America 2024](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/)!
```

{{< image width="75%"
    link="./kubecon-na.png"
    alt="KubeCon + CloudNativeCon North America, November 12-15, 2024, Salt Lake City, Utah. #KubeCon"
    >}}

- Come to the [Istio Day](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/) co-located event.

> - Attend the Istio [Maintainers' Track session - Life of a Packet: Ambient Edition](https://sched.co/1hovw)

Reviewer: ```suggestion
- Attend the Istio Maintainers' Track session: [Life of a Packet: Ambient Edition](https://sched.co/1hovw)
```

> - Attend the Istio [Contribfest session - Sidecarless Service Mesh: Let’s Work Together on Istio V2](https://sched.co/1hoyI)

Reviewer: ```suggestion
- Drop by the Istio Contribfest session: [Sidecarless Service Mesh: Let’s Work Together on Istio V2](https://sched.co/1hoyI)
```

> - The following KubeCon sessions will be based on Istio, add them to your schedule:

Reviewer: ```suggestion
- Add the following KubeCon sessions to your schedule, all of which have an Istio flavor:
```

    - [Why Choose Istio in 2025 | Project Lightning Talk](https://sched.co/1iW9Q)
    - [Lightning Talk: Effortless, Sidecar-Less Mutual TLS and Rich Authorization Policies up and Running in 5 Minutes](https://sched.co/1i7k0)
    - [Poster Session : Unleashing the Power of Prediction to Proactively Scale Control Plane Components](https://sched.co/1i7mr)
    - [What Istio Got Wrong: Learnings from the Last Seven Years of Service Mesh](https://sched.co/1i7nP)
    - [Tutorial: Live with Gateway API V1.2](https://sched.co/1i7np)
    - [`Mish-Mesh`: Abusing the Service Mesh to Compromise Kubernetes Environments](https://sched.co/1i7ow)
    - [Engaging the KServe Community, The Impact of Integrating Solutions with Standardized CNCF Projects](https://sched.co/1i7r4)
    - [How Google Built a New Cloud on Top of Kubernetes](https://sched.co/1i7pE)
    - [Securing Outgoing Traffic: Building a Powerful Internet Egress Gateway for Reliable Connectivity](https://sched.co/1i7ps)
    - [Testing Kubernetes Without Kubernetes: A Networking Deep Dive](https://sched.co/1i7qh)
    - [How GoTo Financial Automates Upgrading 60+ Istio Service Mesh Seamlessly!](https://sched.co/1i7rH)

> - Come and have a chat at the Istio kiosk in the Project Pavilion throughout the event, and grab some cool Istio T-shirts with our brand new design. We also have an interesting surprise for all Istio lovers, to be released at the KubeCon North America CNCF store. Stay tuned!

Reviewer: ```suggestion
- Have a chat with maintainers and users at the Istio kiosk in the Project Pavilion throughout the event, where you can grab a cool Istio T-shirt with our brand new design. 
- We also have an interesting surprise for all Istio lovers, to be released at the KubeCon North America CNCF store. Stay tuned!
```

> Follow us on our social media handles to get live updates from the event. See you soon!

Reviewer: ```suggestion
Follow us on [X](https://x.com/istiomesh), [LinkedIn](https://www.linkedin.com/company/istio/) or [Bluesky](https://bsky.app/profile/istio.io) to get live updates from the event. See you soon!
```
